.github/workflows/build_wheels.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         path: ./dist
 
     - name: Deploy on Test PyPi
-      uses: pypa/gh-action-pypi-publish@v1.8.14
+      uses: pypa/gh-action-pypi-publish@v1.9.0
```

Comment:
@Myoldmopar dependabot is up and running it seems.

Changelog seems to indicate this bump is safe, as changes are minimal and should be non breaking

https://github.com/pypa/gh-action-pypi-publish/releases/tag/v1.9.0